src/grinder/models.py

```diff
@@ -1,3 +1,91 @@
 from django.db import models
-
+from django.contrib.postgres.fields import ArrayField
+from django.db.models import JSONField
 # Create your models here.
+from django.contrib.auth.models import AbstractUser
+
+#    "Whether you want to uncover the secrets of the universe, or you want to pursue a career in the 21st century, basic computer programming is an essential skill to learn."
+#                                                                           ~Stephen Hawking~
+
+class User (AbstractUser):
+    pass
+    access_token = models.CharField(blank=True, default= None, max_length = 200, null=True)
+    refresh_token = models.CharField(blank=True, default= None, max_length = 200, null=True)
+    expires_in = models.IntegerField(blank=True, default= None, null=True)
+
+    def __str__(self):
+        return self.username
+
+    def spRefresh (access_token, refresh_token, expires_in):
+        self.access_token = access_token
+        self.refresh_token = refresh_token
+        self.expires_in = expires_in
```

Comment:
we really need to find a way to apply these updates to database in one go. And I think I found it. you will need to call `save` on object

```suggestion
        self.access_token = access_token
        self.refresh_token = refresh_token
        self.expires_in = expires_in
        self.save()
```